keras_cv/visualization/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib.pyplot as plt
+
+def visualize_grid(ds, figsize=(8, 8)):
+     images, _ = next(iter(ds.take(1)))
```

Comment:
In the case of visualizing an unsupervised dataset, can we add support for that?

Reply:
We can use a boolean positional argument, `title` . For example 

```
if title:
   plt.title(...)
```